applications/GeoMechanicsApplication/custom_processes/periodic_interface_process.hpp

```diff
@@ -55,46 +51,33 @@ class PeriodicInterfaceProcess : public Process
         // Now validate agains defaults -- this also ensures no type mismatch
         rParameters.ValidateAndAssignDefaults(default_parameters);
 
-        mDimension = rParameters["dimension"].GetInt();
+        mDimension   = rParameters["dimension"].GetInt();
         mStressLimit = rParameters["stress_limit"].GetDouble();
 
         KRATOS_CATCH("");
     }
 
-    ///------------------------------------------------------------------------------------
-
-    /// Destructor
-    ~PeriodicInterfaceProcess() override {}
-
-///----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-
-    /// Execute method is used to execute the PeriodicInterfaceProcess algorithms.
-    void Execute() override
-    {
-    }
+    PeriodicInterfaceProcess(const PeriodicInterfaceProcess&) = delete;
+    PeriodicInterfaceProcess& operator=(const PeriodicInterfaceProcess&) = delete;
+    ~PeriodicInterfaceProcess() override = default;
 
     /// this function is designed for being called at the beginning of the computations
     /// right after reading the model and the groups
     void ExecuteInitialize() override
     {
         KRATOS_TRY
 
-        if (mrModelPart.NumberOfConditions() > 0) {
-            block_for_each(mrModelPart.Conditions(), [&](Condition& rCondition) {
-                Condition::GeometryType& rGeom = rCondition.GetGeometry();
+        block_for_each(mrModelPart.Conditions(), [&](Condition& rCondition) {
+            Condition::GeometryType& rGeom = rCondition.GetGeometry();
 
-                rCondition.Set(PERIODIC,true);
+            rCondition.Set(PERIODIC,true);
 
-                rGeom[0].FastGetSolutionStepValue(PERIODIC_PAIR_INDEX) = rGeom[1].Id();
-                rGeom[1].FastGetSolutionStepValue(PERIODIC_PAIR_INDEX) = rGeom[0].Id();
-            });
-        }
-
-        if (mrModelPart.NumberOfElements() > 0) {
-            block_for_each(mrModelPart.Elements(), [&](Element& rElement) {
-                rElement.Set(ACTIVE,false);
-            });
-        }
+            rGeom[0].FastGetSolutionStepValue(PERIODIC_PAIR_INDEX) = rGeom[1].Id();
+            rGeom[1].FastGetSolutionStepValue(PERIODIC_PAIR_INDEX) = rGeom[0].Id();
+        });
+        block_for_each(mrModelPart.Elements(), [&](Element& rElement) {
+            rElement.Set(ACTIVE,false);
+        });
```

Comment:
I recently learned that we can simplify this even further to:
```suggestion
        VariableUtils{}.SetFlag(ACTIVE, false, mrModelPart.Elements());
```

(Note that this requires `#include "utilities/variable_utils.h"`.)

Reply:
Done